flatliners/prometheusendpoint.py

```diff
@@ -0,0 +1,57 @@
+from time import sleep, time
+from collections import defaultdict
+from .baseflatliner import BaseFlatliner
+import dateparser
+import os
+
+from prometheus_client import Gauge, start_http_server
+
+# Set up logging
+import logging
+_LOGGER = logging.getLogger(__name__)
+
+class PrometheusEndpoint(BaseFlatliner):
+    def __init__(self):
+        super().__init__()
+        # Published Stale metrics are removed once every three times metric data is collected from prometheus
+        self.pruning_interval = os.getenv("FLT_METRIC_CHUNK_SIZE","5m")
+        self.pruning_interval = 3 * round((dateparser.parse('now')-dateparser.parse(self.pruning_interval)).total_seconds())
```

Comment:
why `now`? 
don't you just want to convert `5m` to seconds?

Reply:
Yeah that's what it does. `dateparser.parse('5m')` gives a datetime object for 5 minutes earlier than the current datetime.

Reply:
The reason I am using dateparser is because it will be able to process '5m', '5min', '5 minutes' and many other variations for it